.github/workflows/static.yml

```
@@ -25,6 +25,7 @@ permissions:
   contents: write
 env:
   CURRENT_VERSION: "2.2"
```

Lets bump this so we keep the old version of the docs as well.

```suggestion
  CURRENT_VERSION: "2.3"
```